### PR TITLE
[docs][ybm] New release tracks

### PR DESCRIPTION
**Pull request by ddhodge**
Innovation and Production release tracks for dedicated clusters
CLOUDGA-14651

@netlify /preview/faq/yugabytedb-managed-faq/#what-version-of-yugabytedb-does-my-cluster-run-on

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-guide-api.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ To create your free [sandbox](../../../cloud-basics/create-clusters/create-clust
 
 ### Determine the database version
 
-YugabyteDB Managed has [two release tracks](../../../../faq/yugabytedb-managed-faq/#what-version-of-yugabytedb-does-my-cluster-run-on), Preview for testing new features, and Stable for production deployments.
+YugabyteDB Managed has [three release tracks](../../../../faq/yugabytedb-managed-faq/#what-version-of-yugabytedb-does-my-cluster-run-on), Preview and Innovation for testing new features, and Production for production deployments.
```
Comment (daniel-yb) [inline docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-guide-api.md:50]: @shreyas-agnihotri wording wise -- are we okay framing Innovation track as something not suitable for production use cases?

File: docs/content/preview/faq/yugabytedb-managed-faq.md [hunk 2]
```diff
@@ -138,7 +138,7 @@ To keep up with the latest bug fixes, improvements, and security fixes, Yugabyte
 
 Yugabyte only upgrades clusters during scheduled maintenance windows. Yugabyte notifies you in advance of any upcoming upgrade via email. The email includes the date and time of the maintenance window. An Upcoming Maintenance badge is also displayed on the cluster. You can start the upgrade any time by signing in to YugabyteDB Managed, selecting the cluster, clicking the **Upcoming Maintenance** badge, and clicking **Upgrade Now**. To delay the maintenance, click **Delay to next available window**. To manage maintenance windows, select the cluster [Maintenance tab](../../yugabyte-cloud/cloud-clusters/cloud-maintenance/).
 
-The database is upgraded to the latest release in the [release track](#what-version-of-yugabytedb-does-my-cluster-run-on) that was selected when the cluster was created (either preview or stable). Sandbox clusters are always in the preview track.
+The database is upgraded to the latest release in the [release track](#what-version-of-yugabytedb-does-my-cluster-run-on) that was selected when the cluster was created (either Innovation or Production, or Preview for Sandbox clusters). Sandbox clusters are always in the preview track.
```
Comment (shreyas-agnihotri) [inline docs/content/preview/faq/yugabytedb-managed-faq.md:141]: Can we rephrase this to avoid confusion? I missed the comma and read this as: Innovation or Production or Preview for Sandbox clusters.

File: docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-guide-api.md (hunk 1 above)
Comment (shreyas-agnihotri) [reply docs/content/preview/yugabyte-cloud/managed-automation/managed-cli/managed-cli-examples/managed-guide-api.md:50]: No, we need to update this.

@ddhodge Can you check the Release Track Descriptions in this doc (at the end) for reference?
https://docs.google.com/document/d/1uaksIU2quGOSRkFb0k-XnkNGgcoQEJObUGEElSuA6is/edit#heading=h.rwrsy7jxymxq

**Review by aishwarya24 — APPROVED** [review]
LGTM
